local-cli/runIOS/runIOS.js

```diff
@@ -27,15 +27,24 @@ function runIOS(argv, config) {
 }
 
 function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([{
-    command: 'simulator',
-    description: 'Explicitly set simulator to use',
-    type: 'string',
-    required: false,
-    default: 'iPhone 6',
-  }], argv);
+  const args = parseCommandLine([
+    {
+      command: 'simulator',
+      description: 'Explicitly set simulator to use',
```

Comment:
Please stick to the conventions used across the project, for example "Simulator to use, for example 'iPhone 6'."

Also, is this only a simulator, or also possibly a physical device?

See e.g. https://github.com/facebook/react-native/blob/3a92f2017fc2e51c2a915f0e535397cb3937d412/local-cli/generate/generate.js